cmd/kubelet/app/server.go

```diff
@@ -709,6 +732,7 @@ func createAndInitKubelet(kc *KubeletConfig) (k KubeletBootstrap, pc *config.Pod
 	pc = makePodSourceConfig(kc)
 	k, err = kubelet.NewMainKubelet(
 		kc.Hostname,
+		kc.NodeName,
```

Comment:
need similar change in `contrib/mesos/pkg/executor/service/service.go` call to `NewMainKubelet`